kyuubi-server/src/main/scala/org/apache/kyuubi/engine/spark/SparkProcessBuilder.scala

```diff
@@ -127,7 +127,7 @@ class SparkProcessBuilder(
       buffer += CONF
       buffer += s"$k=$v"
     }
-    // iff the keytab is specified, PROXY_USER is not supported
+    // if the keytab is specified, PROXY_USER is not supported
```

Comment:
the old comment is right, iff means only if

Reply:
Get it